functions/nodejs-apps.sh

```diff
@@ -2,14 +2,24 @@
 
 nodejs_setup() {
   if ! command -v npm &>/dev/null; then
-    echo -n "$(timestamp) [openHABian] Installing Node.js (prerequisite for other packages)... "
     FAILED=0
-    cond_redirect wget -O /tmp/nodejs-v7.x.sh https://deb.nodesource.com/setup_7.x || FAILED=1
-    cond_redirect bash /tmp/nodejs-v7.x.sh || FAILED=1
-    if [ $FAILED -eq 1 ]; then echo "FAILED (nodejs preparations)"; exit 1; fi
-    #cond_redirect apt update # part of the node script above
-    cond_redirect apt -y install nodejs
-    if [ $? -ne 0 ]; then echo "FAILED (nodejs installation)"; exit 1; fi
+    if is_pione || is_pizero || is_pizerow; then
```

Comment:
I don't like this condition. Would you please add a is_armv6l function here: https://github.com/openhab/openhabian/blob/master/functions/helpers.sh#L70